CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format of this document is based on [Keep a Changelog](https://keepachangelo
 - The controlboard and multicamera plugins now handle the `yarpDeviceName` parameter (https://github.com/robotology/gazebo-yarp-plugins/pull/559).
 
 ### Changed
+- `MultiCamera`, `LaserSensor`, `DoubleLaser`, `ControlBoard` and `DepthCamera` logging now have a component and they're all yC...() style  
```

Comment:
The CHANGELOG is a document that is also targeted to external users, so it is tipically better to:
* Use non-ambigous names, so for plugins refer to the plugin name that is used in SDF (see https://github.com/robotology/gazebo-yarp-plugins/blob/master/doc/embed_plugins.md#plugins-exposing-functionalities-available-on-real-robots)
* Be clear (what does mean "logging now have a component"?) 

Furthermore, tipically sentences end with a period. I tried to revise the comment considerin this, let me know if you like it. 
```suggestion
- `gazebo_yarp_multicamera`, `gazebo_yarp_lasersensor`, `gazebo_yarp_doublelaser`, `gazebo_yarp_controlboard` and `gazebo_yarp_depthCamera` plugins now log messages using the ["Log Components" YARP logging feature](http://www.yarp.it/git-master/yarp_logging.html).
```

Reply:
Yes you're riight, I'm beginning to learn to do it :) I think the way you've done it is very good, thanks!